types/tengitsui/index.d.ts

```diff
@@ -0,0 +1,332 @@
+// Type definitions for tengitsui 4.4
```

Comment:
It looks like this version doesn't match any version on npm, did you mean 4.3 or is there a reason for this to be 4.4?